### PR TITLE
chore: jq write remove sort

### PR DESCRIPTION
**Pull request by eecsliu**
## Description
Small refactor that gets rid of the sortTasks function

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: master/internal/db/postgres_tasks_intg_test.go [hunk 1]
```diff
@@ -10,12 +10,13 @@ import (
 	"time"
 
 	"github.com/davecgh/go-spew/spew"
-	"github.com/determined-ai/determined/master/pkg/etc"
-	"github.com/determined-ai/determined/master/pkg/model"
-	"github.com/determined-ai/determined/master/pkg/ptrs"
 	"github.com/google/uuid"
 	"github.com/stretchr/testify/require"
 	"gopkg.in/guregu/null.v3"
+
```
Comment (hamidzr) [inline master/internal/db/postgres_tasks_intg_test.go:16]: take out? I think @stoksc  is working on fixing this

**Review by hamidzr — APPROVED** [review]
lgtm. excluding the probably unintentional addition

File: master/internal/db/postgres_tasks_intg_test.go (hunk 1 above)
Comment (hamidzr) [reply master/internal/db/postgres_tasks_intg_test.go:16]: although if this is not tripping up the CI could we just commit this to master? cc @stoksc

**Comment by hamidzr** [comment]
I'm not sure anymore which way the imports should be. Let's stick with whatever looks like master

File: master/internal/db/postgres_tasks_intg_test.go (hunk 1 above)
Comment (stoksc) [reply master/internal/db/postgres_tasks_intg_test.go:16]: I was about to commit it also, idc who does it

Comment (eecsliu) [reply master/internal/db/postgres_tasks_intg_test.go:16]: I might... this was what happened when I did `make fmt` though.

**Pull request merged**